src/olympia/addons/templates/addons/impala/developers.html

```diff
@@ -99,9 +99,9 @@ <h1 class="addon"{{ addon.name|locale_html }}>{{ title }}</h1>
           {{ author|user_vcard }}
         </div>
         <div class="prose">
-          {% if author.bio %}
+          {% if author.biography %}
             <h3>{{ _('Who is {0}?')|f(author.name) }}</h3>
-            <p class="bio">{{ author.bio|nl2br }}</p>
+            <p class="biography">{{ author.biography|nl2br }}</p>
```

Comment:
I suspect the `|no_links` was added to the original template because when the no links policy was introduced we still had bios with links for a short while.